From 2ca5af9cbc92a1c9b884e57e36b445e22b2dd410 Mon Sep 17 00:00:00 2001
From: Jeff Kudrick <jeff.kudrick@freescale.com>
Date: Wed, 27 Jun 2012 22:44:24 -0500
Subject: [PATCH 050/304] ENGR00210497 : Cannot create shortcut on desktop
 after rotation

The setting 'config_reverseDefaultRotation' in the 'config.xml' overlay
file for the i.MX53 SMD had replaced the default value FALSE with TRUE;
however, this is not correct since our tablet does conform to the normal
logic for applying direction of rotation for an application's requested
orientation.  Removed the override of this setting in our 'config.xml'

Signed-off-by: Jeff Kudrick <jeff.kudrick@freescale.com>
---
 imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
index f531577..782a984 100755
--- a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -59,14 +59,6 @@
         <item>"bnep\\d"</item>
     </string-array>
 
-    <!-- If true, the direction rotation is applied to get to an application's requested
-         orientation is reversed.  Normally, the model is that landscape is
-         clockwise from portrait; thus on a portrait device an app requesting
-         landscape will cause a clockwise rotation, and on a landscape device an
-         app requesting portrait will cause a counter-clockwise rotation.  Setting
-         true here reverses that logic. -->
-    <bool name="config_reverseDefaultRotation">true</bool>
-
     <!-- An Array of "[ConnectivityManager connectionType],
                       [# simultaneous connection types]"  -->
     <string-array translatable="false" name="radioAttributes">
-- 
1.8.0

